DEFAULT_CONFIG.json5

```diff
@@ -79,6 +79,18 @@
       //   // root_ca_certificate_base64: "base64-root-ca-certificate",
       // },
 
+      ////
```

Comment:
Can you please apply the same suggestion than in https://github.com/eclipse-zenoh/zenoh-plugin-ros2dds/pull/220#discussion_r1728946937

Reply:
Sure